plugins/aws-appmesh/e2eTests/e2e_test.go

```diff
@@ -11,6 +11,7 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
+//go:build e2e_test || aws_appmesh
```

Comment:
Why is this || in some, but && in others?

Reply:
if im not mistaken, the old annotation `+build one two` indicates or, while `+build one, two` indicates and

Reply:
Correct, but I was asking why we did it that way to begin with.
@rajal-amzn Do you need these "ecs_serviceconnect" and "aws_appmesh" tags? The e2e targets in the Makefile already run tests in that plugin's specific directory, so I don't know why we need to also have plugin-specific build tags?

Reply:
No. We should be good to remove them.